api/v1alpha1/keepalivedgroup_types.go

```diff
@@ -60,6 +60,9 @@ type KeepalivedGroupSpec struct {
 	// +optional
 	UnicastEnabled bool `json:"unicastEnabled,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// +mapType=granular
+	DaemonsetAnnotations map[string]string `json:"daemonsetAnnotations,omitempty"`
```

Comment:
This property will configure the annotations of the Pods created by the DaemonSet. The name could be understood to configure the annotations of the DaemonSet directly. I think it should have another name.

Reply:
Totally reasonable